Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -35,20 +58,20 @@
 #Event or Type ID used for events testing
 Event_OR_Type_ID = '10'
 
-# Profile image
-# Profile picture for testing
-FILE_PATH = '\\gotrain\pref_photos'
-FILE_NAME = 'profile.jpg'
+# Profile image path
+FILE_PATH_PROFILE = r'..\..\Gordon360\browseable\profile\Default\profile.png'
+FILE_PATH_ID = r'..\..\Gordon360\browseable\profile\Default\ID.png'
 
-# API 
+# API. XOR
```

Comment:
Please clarify this comment.  Explaining that it's for local testing would be good. (ref to README?  Be sure README mentions this.)